source/fab/cli.py

```diff
@@ -61,15 +63,18 @@ def calc_linker_flags():
         # todo: test this and get it running
         # 'ifort': (..., [...])
     }
+
     try:
-        linker, linker_flags = linkers[fc]
-    except KeyError:
-        raise NotImplementedError(f"Fab's zero configuration mode does not yet work with compiler '{fc}'")
+        # Get linker from $LD
+        linker, linker_flags = get_tool(os.environ.get("LD", None))
+    except ValueError:
+        # Get linker from linkers, or else just use $FC
+        linker, linker_flags = linkers.get(os.path.basename(fc), (fc, []))
 
     return linker, linker_flags
 
 
-def cli_fab(folder: Path, kwargs: Optional[Dict] = None):
+def cli_fab(folder: Path = None, kwargs: Optional[Dict] = None):
```

Comment:
Since you're making this argument optional don't forget to type hint it with `Optional[]`.